Cannot find module 'libp2p-crypto'

<!--
Thank you for reporting an issue.

This issue tracker is for bugs and issues found within the JavaScript implementation of IPFS.
If you require more general support please file an issue on our discuss forum. https://discuss.ipfs.io/

Please fill in as much of the template below as you're able.

Version: output of `jsipfs version --all` if using the CLI or `ipfs.version((err, version) => {})` if using the instance.
Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows). If using in a Browser, please share the browser version as well.
Subsystem: if known, please specify affected core module name (e.g Bitswap, libp2p, etc).

If possible, please provide code that demonstrates the problem, keeping it as
simple and free of external dependencies as you are able.
-->

- **Version**: 0.32
- **Platform**: all
- **Subsystem**:

<!-- Bug, Feature, Question, Enhancement, Etc -->
#### Type: 

Bug

<!-- 
One of following:
  Critical - System crash, application panic.
  High - The main functionality of the application does not work, API breakage, repo format breakage, etc.
  Medium - A non-essential functionality does not work, performance issues, etc.
  Low - An optional functionality does not work.
  Very Low - Translation or documentation mistake. Something that won't give anyone a bad day.
-->
#### Severity: 

High

#### Description:

`libp2p-crypto` isn't listed in the `package.json` dependencies of `js-ipfs`, but is used

https://github.com/ipfs/js-ipfs/blob/e7d1270590abb313b7519dd29a43f2a927907653/src/core/index.js#L9

```console
$ node src/cli/bin.js daemon
Initializing daemon...
/Users/oli/Code/ipfs/js-ipfs/node_modules/yargs/yargs.js:1144
      else throw err
           ^

Error: Cannot find module 'libp2p-crypto'

$ node --version
v8.12.0

$ npm --version
6.4.1
```

#### Steps to reproduce the error:

- Pull from `js-ipfs` master, 
- `npm install`
- run `node src/cli/bin.js daemon`

<!--
This is for you! Please read, and then delete this text before posting it.
The js-ipfs issues are only for bug reports and directly actionable features.

Read https://github.com/ipfs/community/blob/master/contributing.md#reporting-issues if your issue doesn't fit either of those categories.
-->

--- a/package.json
+++ b/package.json
@@ -128,7 +128,7 @@
     "libp2p": "~0.23.0",
     "libp2p-bootstrap": "~0.9.3",
     "libp2p-circuit": "~0.2.0",
-    "libp2p-crypto": "^0.13.0",
+    "libp2p-crypto": "~0.13.0",
     "libp2p-floodsub": "~0.15.0",
     "libp2p-kad-dht": "~0.10.1",
     "libp2p-keychain": "~0.3.1",

--- a/package.json
+++ b/package.json
@@ -128,6 +128,7 @@
     "libp2p": "~0.23.0",
     "libp2p-bootstrap": "~0.9.3",
     "libp2p-circuit": "~0.2.0",
+    "libp2p-crypto": "^0.13.0",
     "libp2p-floodsub": "~0.15.0",
     "libp2p-kad-dht": "~0.10.1",
     "libp2p-keychain": "~0.3.1",

